### PR TITLE
Reject all profile approval pending requests if admin marks NOT-MY-USER

### PR DESCRIPTION
**Pull request by vikrantbipin**
Reject all profile approval pending requests if admin marks NOT-MY-USER

### CODE REVIEW

File: src/main/java/org/sunbird/profile/service/ProfileServiceImpl.java [hunk 1]
```diff
@@ -2425,6 +2440,28 @@ public SBApiResponse bulkUploadV2(MultipartFile mFile, String orgId, String chan
 		return response;
 	}
 
+	private boolean rejectProfileApprovalRequestById(String userId) {
+		try {
+			// Fetch pending approval requests for the user
+			List<WfStatusEntity> pendingApprovalRequests = wfStatusEntityRepository.findProfileApprovalRequests(
+					userId, Constants.PROFILE, Constants.SEND_FOR_APPROVAL);
+			if (pendingApprovalRequests.isEmpty()) {
```
Comment (Sahil-tarento) [inline src/main/java/org/sunbird/profile/service/ProfileServiceImpl.java]: Please use the collectionUtils class for checking null

File: src/main/java/org/sunbird/profile/service/ProfileServiceImpl.java [hunk 2]
```diff
@@ -1968,6 +1973,16 @@ public SBApiResponse profileMDOAdminUpdate(Map<String, Object> request, String u
 						existingProfileDetails.put(changedObj, profileDetailsMap.get(changedObj));
 						if (Constants.PROFILE_STATUS.equalsIgnoreCase(changedObj)) {
 							updatedProfileStatus = (String) profileDetailsMap.get(changedObj);
+							if (Constants.NOT_MY_USER.equalsIgnoreCase(updatedProfileStatus)) {
+								boolean isRejected = rejectProfileApprovalRequestById(userId);
+								if (!isRejected) {
+									log.error("Failed to reject profile approval requests for userId: {}", userId);
+									response.getParams().setStatus(Constants.FAILED);
+									response.getParams().setErr("Error while rejecting approval requests");
```
Comment (Sahil-tarento) [inline src/main/java/org/sunbird/profile/service/ProfileServiceImpl.java]: Get the msg from constant files

**Review by Sahil-tarento — CHANGES_REQUESTED** [review]

File: src/main/java/org/sunbird/profile/service/ProfileServiceImpl.java (hunk 1 above)
Comment (vikrantbipin) [reply src/main/java/org/sunbird/profile/service/ProfileServiceImpl.java]: Updated the PR

File: src/main/java/org/sunbird/profile/service/ProfileServiceImpl.java (hunk 2 above)
Comment (vikrantbipin) [reply src/main/java/org/sunbird/profile/service/ProfileServiceImpl.java]: Updated the PR

**Pull request merged**